src/App.scss

```diff
@@ -39,3 +39,9 @@ body {
 a {
   text-decoration: none;
 }
+
+@media (min-width: 1800px) {
```

Comment:
instead of using @media, can you not use material UI's Grid as container and then specify how the display should be have on lg, xl screens?